angularjs/Signature/How-To.md

```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
-title: Syncfusion Signature How To
-description: How To
-platform: AngularJS
+title: How To Section in AngularJS Signature | Syncfusion
+description: Learn here all about how to section Syncfusion AngularJS Signature component, its elements and more.
+platform: angularjs
 control: Signature
 documentation: ug
 ---
 
-## How To?
+# How To Section AngularJS Signature
```

Comment:
Update like this - How To Section in AngularJS Signature

Reply:
updated